.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -85,6 +85,7 @@ body:
       description: Which version of Reserver are you using?
       options:
         - Reserver 0.1
+        - Reserver 0.2
```

Comment:
Put the latest version at the top. Take a look at [Nava's bug_report.yml](https://github.com/openscilab/nava/blob/main/.github/ISSUE_TEMPLATE/bug_report.yml?plain=1) file as an example.